drawBot/drawBotDrawingTools.py

```diff
@@ -2005,6 +2008,8 @@ def imagePixelColor(self, path, xy):
         x, y = xy
         if isinstance(path, str):
             path = optimizePath(path)
+        elif isinstance(path, pathlib.Path):
+            path = f'{path}'
```

Comment:
Here, you should change the previous "if" into `if isinstance(path, (str, os.PathLine))`. `optimizePath()` calls `os.fspath()`, so no further action is needed here.

Reply:
so, if I understand correctly, in every spot where `optimizePath(path)` is called, I can use if `isinstance(path, (str, os.PathLike))`, right?

Reply:
I don't know about that, but in this place, yes.

Reply:
with "every spot", I meant every spot where I made the edits for this pull request. It's probably easier just to look at the modified code, I'll push it in a moment